data/human/boarding missions.txt

```diff
@@ -703,3 +703,238 @@ mission "Turn in Tarquin"
+				depart
+			label crumple
+			`	As you are patching up a segment of hull you notice that a deformation in it is slowly expanding. To make matters worse, this is a structurally essential part of the hull. You quickly run back to your ship before the <origin> breaks apart.`
+				depart
+			label misfire
+			`	As you are fixing the engine systems a valve bursts open. With no power the secondary valve cannot close itself off, letting coolant uncontrollably fly into the hull. The force of the liquid sends a violent vibration shuddering through the ship. You quickly run back to your ship before the <origin> breaks apart.`
+				depart
+			label flee
+			action
+				payment 7500
+			`	Wary of your own foolish repairs you decide to leave the ship alone. However, you did manage to hang onto a little bit of scrap worth <payment> you found during your repairs.`
+				depart
+			label fail
+			`	As it begins to thrum to life, a loud explosion rocks the ship!`
+			choice
+				`	(Run.)`
+			`	You start running through the tight corridors of the ship. Objects are tumbling everywhere, and more explosions threaten to sweep you off your feet.`
+			choice
+				`	(Run into the airlock.)`
+					goto run
+				`	(Dive for the airlock.)`
+					goto dive
+				`	(Slide into the airlock.)`
+			`	As you slide across the floor, explosions erupt on both sides of you, and you're sure that you look like an action hero as you careen through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label run
+			`	As you run towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you barge through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label dive
+			`	As you dive towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you tuck and roll through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label success
+			branch beacon
+				random < 15
+			branch virus
+				random < 5
+				"credits" > 1000000
+			action
+				payment 65000
+			`	Despite your best efforts to make the ship actually fly, it just won't. On the upside, now you have time to loot and pilfer some outfits from the ship as you please. Before leaving the cockpit you find a credit chip sitting in the cockpit worth <payment>.`
+				decline
+			label beacon
+			action
+				set "derelict pirate beacon"
+			`	However, you soon notice a blinking light on the controls of the ship. You wonder what it could signal as you approach it, and once you get close enough you realize that it warns of the arrival of 'hostile spacecraft' entering the system. You now have a pretty good guess at what you're dealing with as you race back to your ship.`
```

Comment:
```suggestion
			`	However, you soon notice a blinking light on the controls of the ship. You wonder what it could signal as you approach it, and once you get close enough you realize that it warns of the arrival of "hostile spacecraft" entering the system. You now have a pretty good guess at what you're dealing with as you race back to your ship.`
```
The ship will explode as a result of this, which isn't explained at all. The way this is described just seems like the hostile radar sound that the player's ship can make (given that you have that setting active) if a hostile enters the system, but the way that it actually plays out is more like the ship is booby trapped.